frontend/src/e2e-test/specs/5_employee/child-discussion.spec.ts

```diff
@@ -0,0 +1,162 @@
+// SPDX-FileCopyrightText: 2017-2022 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import LocalDate from 'lib-common/local-date'
+
+import config from '../../config'
+import {
+  insertDaycareGroupFixtures,
+  insertDaycarePlacementFixtures,
+  resetDatabase
+} from '../../dev-api'
+import { initializeAreaAndPersonData } from '../../dev-api/data-init'
+import {
+  createDaycarePlacementFixture,
+  daycareGroupFixture,
+  Fixture,
+  uuidv4
+} from '../../dev-api/fixtures'
+import {
+  DaycarePlacement,
+  EmployeeDetail,
+  PersonDetailWithDependantsAndGuardians
+} from '../../dev-api/types'
+import ChildInformationPage, {
+  ChildDocumentsSection
+} from '../../pages/employee/child-information'
+import { Page } from '../../utils/page'
+import { employeeLogin } from '../../utils/user'
+
+let page: Page
+let admin: EmployeeDetail
+let childInformationPage: ChildInformationPage
+let child: PersonDetailWithDependantsAndGuardians
+let daycarePlacementFixture: DaycarePlacement
+
+beforeAll(async () => {
+  await resetDatabase()
+
+  admin = (await Fixture.employeeAdmin().save()).data
+
+  const fixtures = await initializeAreaAndPersonData()
+  await insertDaycareGroupFixtures([daycareGroupFixture])
+
+  const unitId = fixtures.daycareFixture.id
+  child = fixtures.familyWithTwoGuardians.children[0]
+
+  daycarePlacementFixture = createDaycarePlacementFixture(
+    uuidv4(),
+    child.id,
+    unitId
+  )
+
+  await insertDaycarePlacementFixtures([daycarePlacementFixture])
+
+  await Fixture.groupPlacement()
+    .with({
+      daycareGroupId: daycareGroupFixture.id,
+      daycarePlacementId: daycarePlacementFixture.id,
+      startDate: daycarePlacementFixture.startDate,
+      endDate: daycarePlacementFixture.endDate
+    })
+    .save()
+})
+
+describe('Child Information - Child discussion section', () => {
+  let section: ChildDocumentsSection
+  beforeEach(async () => {
+    page = await Page.open({
```

Comment:
Because these tests do stuff with dates, you should consider using `mockedTime` to set the "current time" for the test. This is not strictly necessary in this case because there's no computations or limitations regarding the dates, but it's a good practice nevertheless to make sure that these tests don't break some day in the future when the date is different than it is today.

Reply:
Good point, I will make the change to leverage `mockedTime` instead.